egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -463,10 +467,10 @@ def TrainOneIteration(dir, iter, srand, egs_dir,
     TrainNewModels(dir, iter, srand, num_jobs, num_archives_processed, num_archives,
```

Comment:
Are you suggesting use named arguments for all of them?  if so we might also need to use named arguments for other function calls for the same reason. I think it might not be necessary since this function is called once within only this script.